Testing/ORKTest/ORKTest/MainViewController.m

```diff
@@ -456,8 +458,41 @@ - (UICollectionViewCell *)collectionView:(UICollectionView *)collectionView cell
         return [self makeNavigableOrderedTask];
     } else if ([identifier isEqualToString:CustomNavigationItemTaskIdentifier]) {
         return [self makeCustomNavigationItemTask];
-    }
-    return nil;
+    } if ([identifier isEqualToString:WaitTaskIdentifier]) {
+        
+        NSMutableArray *steps = [[NSMutableArray alloc] init];
+        
+        /*
+         To properly use the wait steps, one needs to implement the "" method of ORKTaskViewControllerDelegate to start their background action when the wait task begins, and then call the "finish" method on the ORKWaitTaskViewController when the background task has been completed.
+         */
+        ORKInstructionStep *step1 = [[ORKInstructionStep alloc] initWithIdentifier:@"waitTask.step1"];
+        step1.title = @"Setup";
+        step1.detailText = @"ORKTest needs to set up some things before you begin, once the setup is complete you will be able to continue.";
+        [steps addObject:step1];
+        
+        // Interterminate wait step.
+        ORKWaitStep *step2 = [[ORKWaitStep alloc] initWithIdentifier:@"waitTask.step2"];
+        step2.title = @"Getting Ready";
+        [steps addObject:step2];
+        
+        ORKInstructionStep *step3 = [[ORKInstructionStep alloc] initWithIdentifier:@"waitTask.step3"];
+        step3.title = @"Account Setup";
+        step3.detailText = @"The information you entered will be sent to the secure server to complete your account setup.";
+        [steps addObject:step3];
+        
+        // Determinate wait step.
+        ORKWaitStep *step4 = [[ORKWaitStep alloc] initWithIdentifier:@"waitTask.step4"];
+        step4.title = @"Syncing Account";
+        step4.indicatorMask = ORKProgressIndicatorMaskProgressBar;
+        [steps addObject:step4];
+        
+        ORKCompletionStep *step5 = [[ORKCompletionStep alloc] initWithIdentifier:@"waitTask.step5"];
+        step5.title = @"Setup Complete";
+        [steps addObject:step5];
+        
+        return [[ORKOrderedTask alloc] initWithIdentifier:WaitTaskIdentifier steps:steps];
+    } else
+        return nil;
```

Comment:
Remove `else` and indent back the `return nil` to stay consistent with the original code for returning nil.